tribe-ext-view-tickets/src/Tribe/Extension/Tickets_Order_Helper2.php

```diff
@@ -1,17 +1,17 @@
 <?php
-// Do not load directly.
-if ( ! defined( 'ABSPATH' ) ) {
-	die( '-1' );
-}
-
-if ( class_exists( 'Tribe__Extension__Tickets_Order_Helper' ) ) {
+/**
+ * Version 2 changelog:
+ * - Fixes a bug where the ticket unique ID was not properly exported
+ * - Allows the passage of attendee or order IDs
+ */
+if ( class_exists( 'Tribe__Extension__Tickets_Order_Helper2' ) ) {
 	return;
 }
 
 /**
  * Helps get the Event IDs, attendees, and ticket provider associated with an order ID
  */
-class Tribe__Extension__Tickets_Order_Helper {
+class Tribe__Extension__Tickets_Order_Helper2 {
```

Comment:
This feels like an odd tactic ... is there a reason the extension (and this class) can't keep the same name as before? Why add `2` - is there something about extensions that means there is a greater risk of users having concurrent versions of the same extension?